bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2556,6 +2557,21 @@ Bool_t PyROOT::Pythonize( PyObject* pyclass, const std::string& name )
          AddArrayInterface(pyclass, (PyCFunction)STLVectorArrayInterface<unsigned long, 'u'>);
       }
 
+      // add array interface for RVecs
+      if (name.find("RVec<") == std::string::npos) {
+      } else if (name.find("<float>") != std::string::npos) {
+         AddArrayInterface(pyclass, (PyCFunction)RVecArrayInterface<float, 'f'>);
+      } else if (name.find("<double>") != std::string::npos) {
+         AddArrayInterface(pyclass, (PyCFunction)RVecArrayInterface<double, 'f'>);
+      } else if (name.find("<int>") != std::string::npos) {
+         AddArrayInterface(pyclass, (PyCFunction)RVecArrayInterface<int, 'i'>);
+      } else if (name.find("<unsigned int>") != std::string::npos) {
+         AddArrayInterface(pyclass, (PyCFunction)RVecArrayInterface<unsigned int, 'u'>);
+      } else if (name.find("<long>") != std::string::npos) {
+         AddArrayInterface(pyclass, (PyCFunction)RVecArrayInterface<long, 'i'>);
+      } else if (name.find("<unsigned long>") != std::string::npos) {
+         AddArrayInterface(pyclass, (PyCFunction)RVecArrayInterface<unsigned long, 'u'>);
+      }
```

Comment:
I wonder if [`std::hash`](https://en.cppreference.com/w/cpp/types/type_index/hash) wouldn't be a better fit here instead of all this string searching on type names...

Reply:
What is the computational cost of `std::hash` vs `std::string::find`? I would prefer performance over "stability" (or "unambiguity") here because this code runs interactively.

Reply:
That is my point, a hash usually costs less than the string search.

Reply:
Hm ok, there's no definite limit for `std::string::find`, but it seems to be in O(n). That should be similar to `std::hash` (which should loop once over the string as well).

Reply:
Ok, we have following case as a bug: Pythonizing the type `RVec<RVec<float>>`, that would fail gloriously with the current implementation. Hashing fixes this nicely. I'll go for it, thanks @amadio!